### PR TITLE
[ES] Update `Help centre/Account restrictions`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Account_restrictions/es.md [hunk 1]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
 
-# Restricciones de la cuenta 
+# Restricciones de la cuenta
+
+*Página principal: [Centro de ayuda](/wiki/Help_centre)*
 
 Una restricción es un tiempo fuera en la comunidad forzada a cuentas que exihiben comportamiento anormal, sospechoso o que rompe las reglas.
 
-Cuando una cuenta es restringida, es incapaz de interactuar con la comunidad hasta que la restricción sea levantada. Los jugadores restringidos pueden seguir jugando el juego, descargar mapas y subir puntuaciones, pero su perfil será invisible para los demás
+Cuando una cuenta es restringida, es incapaz de interactuar con la comunidad hasta que la restricción sea levantada. Los jugadores restringidos pueden seguir jugando el juego, descargar mapas y subir puntuaciones, pero su perfil será invisible para los demás.
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:13]:
```suggestion
Cuando una cuenta es restringida, es incapaz de interactuar con la comunidad hasta que la restricción sea levantada. Los jugadores restringidos pueden seguir jugando el juego, descargar beatmaps y subir puntuaciones, pero su perfil será invisible para los demás.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Account_restrictions/es.md [hunk 2]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
 - Si crees fuertemente que tu restricción fue errónea, por favor contáctenos inmediatamente para que podamos discutir su caso.
 
-Si tu cuenta ha sido restringida por romper las [reglas](/wiki/Rules), un estricto tiempo de espera de 3 meses será forzado antes de que pueda enviar una apelación.
+Si su cuenta ha sido restringida por romper las [reglas](/wiki/Rules), debe esperar un mínimo de **tres meses** antes de poder enviar una apelación. Nosotros nos referimos a este período de tiempo como *cooldown*, y si has hecho trampa, son seis meses.
 
-Si tu cuenta original no está restringida, puedes iniciar sesión con ella y seguir jugando normalmente, pero asegúrate de no iniciar sesión con tus multicuentas en adelante, ya que hacerlo puede dar lugar a un castigo mayor.
+Si su cuenta original no está restringida, puede iniciar sesión con ella y seguir jugando normalmente, pero asegúrate de no iniciar sesión con tus multicuentas en adelante, ya que hacerlo puede dar lugar a un castigo mayor.
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:36]:
```suggestion
Si su cuenta original no está restringida, puede iniciar sesión con ella y seguir jugando normalmente, pero asegúrese de no iniciar sesión con sus multicuentas en adelante, ya que hacerlo puede dar lugar a un castigo mayor.
```

File: wiki/Help_centre/Account_restrictions/es.md [hunk 3]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
-- **Mantén un buen comportamiento.** Entendemos que ser restringido puede ser frustrante, pero tómalo como una oportunidad de tomar un descanso, lejos de la comunidad. Intenta evitar interactuar con la comunidad por lo menos unos cuantos días (ya sea vía Discord, redes sociales, u otras formas de contacto) para respirar y reflexionar las razones por las que su cuenta pudo ser restringida.
-- **Asegúrate de respetar las reglas** Restricciones a largo plazo son castigos por romper las reglas. Si continúas mientras estás restringido, el tiempo de espera para apelar será reiniciado a tres meses desde la última infracción. Esto incluye, por ejemplo, seguir haciendo trampa, accesar a las cuentas de otros, o crear nuevas cuentas para evadir la restricción.
+- **¡Sigue disfrutando del juego!** Mientras tu cuenta está restringida y en cooldown, eres más que bienvenido a seguir jugando el juego (¡siempre y cuando juegues en tu cuenta original!). Las puntuaciones que consigas mientras estás restringido serán subidas, a no ser que la cuenta sea revertida, ellas seguirán después de que la restricción sea levantada.
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:53]:
```suggestion
- **¡Sigue disfrutando del juego!** Mientras su cuenta está restringida y en cooldown, eres más que bienvenido a seguir jugando el juego (¡siempre y cuando juegues en tu cuenta original!). Las puntuaciones que consigas mientras estás restringido serán subidas, a no ser que la cuenta sea revertida, ellas seguirán después de que la restricción sea levantada.
```

File: wiki/Help_centre/Account_restrictions/es.md [hunk 4]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
+- **No dudes en contactarnos si tiene alguna pregunta.** Mientras no intente salvarse del castigo, ¡felizmente contestaremos cualquier pregunta que tenga sobre la restricción de su cuenta!
 
-## Mi tiempo de espera ha terminado. ¿Cómo puedo apelar?  {#appeal}
+## Mi tiempo de espera ha terminado. ¿Cómo puedo apelar? {#appeal}
 
-Puedes apelar enviando un correo a [accounts@ppy.sh](mailto:accounts@ppy.sh) incluyendo el nombre de su cuenta. Asegúrese de enviar el correo desde la dirección de correo asociada a su cuenta. Después de que envíe su apelación, espere una respuesta en un plazo de una semana. Si no has recibido una para ese entonces, por favor contáctenos vía correo o twitter - hay una posibilidad de que no lo hayamos recibido
+Puedes apelar enviando un correo a [accounts@ppy.sh](mailto:accounts@ppy.sh) incluyendo el nombre de su cuenta. Asegúrese de enviar el correo desde la dirección de correo asociada a su cuenta. Después de que envíe su apelación, espere una respuesta en un plazo de una semana. Si no has recibido una para ese entonces, por favor contáctenos vía correo o twitter - hay una posibilidad de que no la hayamos recibido
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:61]:
```suggestion
Puedes apelar enviando un correo a [accounts@ppy.sh](mailto:accounts@ppy.sh) incluyendo el nombre de su cuenta. Asegúrese de enviar el correo desde la dirección de correo asociada a su cuenta. Después de que envíe su apelación, espere una respuesta en un plazo de una semana. Si no has recibido una para ese entonces, por favor contáctenos vía correo o twitter, hay una posibilidad de que no la hayamos recibido
```

File: wiki/Help_centre/Account_restrictions/es.md [hunk 5]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
-- **Cómo conseguiste los medios para hacerlo.** Esto aplica principalmente al hacer trampas - queremos saber dónde conseguiste la herramienta que usaste, cómo la encontraste, y si sigue o no presente en su computador.
-- **Porqué deberíamos permitirte regresar.** La mayor parte del tiempo, esto se enlaza con el segundo punto. Solo queremos asegurarnos de que no romperás las reglas de nuevo! Por favor entienda que entre mas infracciones haya en el historial de su cuenta, más esperaremos esta sección de la apelación
+- **Por qué rompiste las reglas.** Una explicación sobre lo que te llevó a decidir crear mas cuentas, prestar tu cuenta, hacer trampa, etc. nos ayudará (y a ti) a determinar si está en riesgo de infringir las reglas otra vez. Si continuaste infringiendo las reglas mientras estabas restringido, estamos interesados en saber por qué.
+- **Cómo conseguiste los medios para hacerlo.** Esto aplica principalmente al hacer trampas - queremos saber dónde conseguiste la herramienta que usaste, cómo la encontraste, y si sigue o no presente en su computadora.
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:67]:
```suggestion
- **Cómo conseguiste los medios para hacerlo.** Esto aplica principalmente al hacer trampas, queremos saber dónde conseguiste la herramienta que usaste, cómo la encontraste, y si sigue o no presente en su computadora.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Account_restrictions/es.md [hunk 6]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
+Si su cuenta ha sido restringida, no entre en pánico. Por favor sigue los siguientes pasos:
 
-- **Espera 24 horas.** Algunas restricciones son resultado de procesos automatizados. Como revisamos cada nueva restricción diariamente, por favor espere 24 horas - si ha sido restringido erróneamente, probablemente la restricción será levantada en ese tiempo.
-- **Contacte a nuestro [equipo de atención a cuentas](/wiki/People/Account_support_team).** Si su cuenta sigue restringida después de ese tiempo, por favor envíe un correo electrónico a [accounts@ppy.sh](mailto:accounts@ppy.sh) **desde el correo electrónico ligado a su cuenta** para más detalles de su restricción. Asegúrate de agregar el nombre de tu cuenta, junto con cualquier detalle que nos pueda proveer sobre razones por las que pudo ser restringido. Entre más detalles nos cuente, más fácil será para nosotros resolver el problema.
+- **Esperar 24 horas.** Algunas restricciones son resultado de procesos automatizados. Como revisamos cada nueva restricción diariamente, por favor espere 24 horas - si ha sido restringido erróneamente, probablemente la restricción será levantada en ese tiempo.
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:30]:
```suggestion
- **Esperar 24 horas.** Algunas restricciones son resultado de procesos automatizados. Como revisamos cada nueva restricción diariamente, por favor espere 24 horas, si ha sido restringido erróneamente, probablemente la restricción será levantada en ese tiempo.
```

File: wiki/Help_centre/Account_restrictions/es.md [hunk 7]
```diff
@@ -4,116 +4,130 @@ outdated_translation: true
...
 
 Por favor tome en cuenta que enviar un correo a esta dirección solo garantiza un segundo vistazo/inicial respuesta a tu apelación, no garantiza un cambio en el resultado si te han encontrado violando las expectativas involucradas en el proceso de apelación.
 
 **Solo responderemos a solicitudes para revisión de apelaciones/investigaciones vía correos enviados a [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Por favor no intente contactar a los miembros del equipo de soporte (o miembros del equipo de voluntarios) vía Twitter, Discord, o cualquier otra plataforma para evitar esta política - ellos no pueden ayudarte.
 
-## Razones comunes de restricción y tiempos de espera  {#reasons}
+## Razones comunes de restricción y tiempos de espera {#reasons}
 
-| Razón de restricción | Duración del TdE | Notas de apelación |
+| Razón de restricción | Duración del Tiempo de Espera | Notas de apelación |
```
Comment (agatemosu) [inline wiki/Help_centre/Account_restrictions/es.md:122]:
```suggestion
| Razón de restricción | Duración del tiempo de espera | Notas de apelación |
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by TPGPL — DISMISSED** [review]
translation is outdated, please apply https://github.com/ppy/osu-wiki/pull/9200

**Pull request merged**